modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -382,6 +382,97 @@ definition {
 		}
 	}
 
+	@description = "LPS-165849 - Verify if terms from a related custom object (child) with a system object (parent) are replaced correctly in an email notification"
+	@priority = 5
+	test CanSendEmailNotificationWithTermsComingFromSystemObjectRelationship {
+		property custom.properties = "feature.flag.LPS-165849=true";
+		property portal.acceptance = "true";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given an object with a field is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 182606",
+				objectName = "CustomObject182606",
+				pluralLabelName = "Custom Objects 182606");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Field Text",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject182606");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject182606");
+		}
+
+		task ("And given the User object with a 1:M relationship to custom object") {
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "User",
+				objectName_2 = "CustomObject182606",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+		}
+
+		task ("And given a email notification template using terms from both object definitions") {
```

Comment:
And given an email notification template using terms from both object definitions